lib/my_inject.rb

```diff
@@ -0,0 +1,41 @@
+class Array
+	
+	def my_inject(accumulator = nil, operation = nil, &block)
```

Comment:
Assignment Branch Condition size for my_inject is too high. [21.21/15]<br>Use 2 (not 1) spaces for indentation.<br>Tab detected.